grails-app/controllers/org/broadinstitute/orsp/AuthenticatedController.groovy

```diff
@@ -62,7 +62,21 @@ class AuthenticatedController implements Interceptor, UserInfo {
 
     protected Collection<User> getPIsForIssue(Issue issue) {
         Collection<String> pis = IssueExtraProperty.findAllByProjectKeyAndName(issue.projectKey, IssueExtraProperty.PI)*.value
-        userService.findUsers(pis)
+        Collection<User> pisForUsers = userService.findUsers(pis) ?: new ArrayList<>()
```

Comment:
`findUsers` will *always* return a list, either a list of users (which should be immutable, but it's a GORM thing and it might be mutable) or an immutable empty list. You could run into trouble trying to modify it below if it does return an empty list.

Reply:
Ah, I see, i created a new collection to store the info returned by findUsers, and prevent that trouble, thanks for the advice!